src/dns/error.cpp

```diff
@@ -0,0 +1,86 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <measurement_kit/common/error.hpp>
+#include <measurement_kit/dns/error.hpp>
+
+#include <event2/dns.h>
+
+namespace measurement_kit {
+namespace dns {
+
+common::Error dns_error(int code) {
```

Comment:
I understand now why you removed it from the above. Still perhaps it makes sense to have the function inside of the namespace of the class, though it's also a bit bikeshedding.

I will think more about this too.